addons/account/models/chart_template.py

```diff
@@ -149,7 +149,7 @@ def try_loading(self, template_code, company, install_demo=True):
 
         template_code = template_code or company and self._guess_chart_template(company.country_id)
 
-        if template_code == 'syscohada' and template_code != company.chart_template:
+        if template_code in ('syscohada', 'syscebnl') and template_code != company.chart_template:
```

Comment:
Should we modified the user error aswell (and by doing so the translation) ?

Reply:
Im not sure but I can edit it. Both charts apply to the OHADA countries and are part of the syscohada module

Reply:
done, let me know if this is the correct way to update translations